### PR TITLE
fix[s3]: add required parameter region to migrator

### PR DESCRIPTION
**Pull request by haooliveira84**
### Description of your changes

Fixes #
Add required parameter `region` in S3

Ref:
https://marketplace.upbound.io/providers/upbound/provider-aws-s3/v0.42.0/resources/s3.aws.upbound.io/Bucket/v1beta1#doc:spec-forProvider-region

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: .vscode/settings.json
Comment (sergenyalcin) [inline .vscode/settings.json:1]: I think this file was not  added intentionally.

File: pkg/converter/configuration/converter.go
Comment (sergenyalcin) [inline pkg/converter/configuration/converter.go:1]: I think this branch uses #37 as base. So we may consider handling this PR after merging the #37

**Review by sergenyalcin — COMMENTED** [review]
Thanks @haooliveira84 I left a few comments

File: .vscode/settings.json
Comment (haooliveira84) [reply .vscode/settings.json:1]: removed

File: pkg/converter/configuration/converter.go
Comment (haooliveira84) [reply pkg/converter/configuration/converter.go:1]: done

**Review by sergenyalcin — APPROVED** [review]
Thanks @haooliveira84 LGTM!

**Pull request merged**